bot/constants.py

```diff
@@ -1,10 +1,26 @@
 import pathlib
 import os
+import typing
 
 # env vars
 PREFIX = os.getenv("PREFIX") or "!"
 TOKEN = os.getenv("TOKEN")
 
+
+class Roles(typing.NamedTuple):
+    ANNOUNCEMENTS_ID = int(os.getenv("ANNOUNCEMENTS_ID", "789978290844598272"))
```

Comment:
I don’t see a need to make them upper cased as they are already in a constants.py file
```suggestion
    announcements = int(os.getenv("ANNOUNCEMENTS_ID", "789978290844598272"))
```